packages/aws-client/tests/test-StepFunctions.js

```diff
@@ -164,3 +201,72 @@ test('executionExists returns false if the execution does not exist', async (t)
   const executionArn = 'arn:aws:states:us-east-1:123456789012:execution:MyStackIngestAndPublishGranuleStateMachine:c154d37a-98e5-4ca9-9653-35f4ae9b59d3';
   t.false(await StepFunctions.executionExists(executionArn));
 });
+
+test('getExecutionStatus() throws exception if the execution does not exist', async (t) => {
```

Comment:
Are these new tests?

Reply:
Yes, I added these unit tests to provide 100% coverage for StepFunctions.ts. Is that ok?

Reply:
That's great! Just went over them. Thanks for doing that!